core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -236,6 +264,16 @@ func (i *pluginOracleCreator) createFactoryAndTransmitter(
 		return nil, nil, fmt.Errorf("unsupported chain selector %d %w", config.Config.ChainSelector, err)
 	}
 
+	chainFamily, err := chainsel.GetSelectorFamily(uint64(config.Config.ChainSelector))
+	if err != nil {
+		return nil, nil, fmt.Errorf("unsupported chain selector %d %w", config.Config.ChainSelector, err)
+	}
+	plugin, exists := plugins[chainFamily]
+	if !exists {
+		return nil, nil, fmt.Errorf("unsupported chain %v", chainFamily)
+	}
+	messageHasher := plugin.MessageHasher(i.lggr.Named("MessageHasherV1").With("chainFamily", chainFamily))
```

Comment:
Small consistency nit

```suggestion
	messageHasher := plugin.MessageHasher(i.lggr.Named(fmt.Sprintf("%s.MessageHasherV1", chainFamily)))
```

The reasoning behind this is that the chainlink node component loggers are all like this e.g `EVM.1337.LogPoller`, etc.

Reply:
Probably better to call named twice instead of assuming the separator:
```suggestion
	messageHasher := plugin.MessageHasher(i.lggr.Named(chainFamily).Named("MessageHasherV1"))
```
We also typically say "network" instead of chain family.

Reply:
I can make this change but I switched to using a tag based on a previous review https://github.com/smartcontractkit/chainlink/pull/15820#discussion_r1905816002